design/views/empty states.mdx

```diff
@@ -0,0 +1,42 @@
+---
+name: Empty states
+menu: Views
+---
+
+# Empty states
+
+## Default empty states
+
+### When there are no workloads
+
+![no workloads](../../sketch-images/When-there-are-no-workloads.png)
+
++ Use this empty state across workloads when there are no items to show. Include the resource’s schema description and link to the API Explorer to learn more. The primary action should allow the user to Create [Resource].
++ In cases where users should not be creating objects in specific namespaces, remove the primary Create button and include informational text in the body explaining why.
++ Where applicable, use the resource’s badge in grey for the empty state icon.
+
+![no workloads2](../../sketch-images/When-there-are-no-workloads2.png)
+
++ For this case, the primary Create button should be “Add IDPs” and redirect users to the OAuth page. A user icon should be used for this empty state.
+
+![no workloads3](../../sketch-images/When-there-are-no-workloads3.png)
+
++ Since users cannot create or add events, this empty state does not have a primary action.
+
+### Could not connect
+
+![could not connect](../../sketch-images/Could-not-connect.png)
+
+### When nothing matches a filter
+
+![nothing matches a filter connect](../../sketch-images/When-nothing-matches-a-filter.png)
+
+---
+
+## Small empty states
+
+The following empty states require minimal updates to font size and color but are otherwise the same as how they appear in the console today.
+
+![small empty states1](../../sketch-images/Small-empty-state.png)
```

Comment:
What do you think about cropping this screenshot in order to better highlight the empty state? Maybe just show Containers, Volumes, and Conditions sections and crop out the Overview?